dynamic_rest/templates/dynamic_rest/horizontal/select2.html

```diff
@@ -0,0 +1,80 @@
+{% load i18n %}
```

Comment:
The only real template change: relationship fields based on select2 with query back to API